libraries/AP_Notify/Buzzer.cpp

```diff
@@ -68,15 +68,57 @@ void Buzzer::update()
 
 void Buzzer::update_pattern_to_play()
 {
+    if (!_flags.initialize_done) {
+        // play a quick beep at startup
+        if (!_flags.startup_beep) {
+            _flags.startup_beep = true;
+            play_pattern(SINGLE_BUZZ);
+            return;
+        }
+        // play beeps when system initialization complete
+        if (!_flags.initialize_started) {
```

Comment:
We should be consistent within a file in terms of spelling.  In this case we're (sensibly!) using `s` to produce a `z` sound, so we should do that in changes to the file.

Reply:
Sure, no problem.